### PR TITLE
R4R: Update gov and snapshot docs

### PR DESCRIPTION
**Pull request by zhangyelong**

### CODE REVIEW

File: docs/daemon/snapshot.md [hunk 1]
```diff
@@ -38,3 +38,19 @@ Snapshot current node's latest data
...
  iris snapshot --home=<path-to-your-home>
 ```
+
+## FAQ
+
+### What happens if everyone delete previous stored blockchain data and a new machine will try to sync up
+
+If that happens, a new machine will be unable to sync from scratch, but it can use a snapshot to catch up faster.
+
+I think some of the nodes will not delete all the previous data, such as nodes for explorers and wallets. And we, the IRIS Foundation will keep all the data too, also we can offer a full data snapshot. And we encourage snapshot service providers could provide both the minimal and the full snapshot :)
```
Comment (chengwenxi) [inline docs/daemon/snapshot.md]: we should avoid using `I`. (just I think)

Comment (haifengxi) [reply docs/daemon/snapshot.md]: Agreed.  "Presumably not all nodes will delete historical data, ..."

File: docs/daemon/snapshot.md [hunk 2]
```diff
@@ -38,3 +38,19 @@ Snapshot current node's latest data
...
+
+### But this will have a slow speed of download probably
+
+Correct, but most people don't need the full data, they can download a latest snapshot to sync up much faster than before. If they want, they can also download the full data snapshot too.
+
+### Can I snapshot the validator node
+
+Yes, but better not. Time will tell if we can be brave enough to do that :)
```
Comment (haifengxi) [inline docs/daemon/snapshot.md]: Yes, but we wouldn't recommend that for the time being.

File: docs/daemon/snapshot.md [hunk 3]
```diff
@@ -38,3 +38,19 @@ Snapshot current node's latest data
 ```bash
  iris snapshot --home=<path-to-your-home>
 ```
+
+## FAQ
+
+### What happens if everyone delete previous stored blockchain data and a new machine will try to sync up
```
Comment (haifengxi) [inline docs/daemon/snapshot.md]: What if every node in the network runs from a snapshot and a new node tries to catch up from genesis

**Pull request merged**